esmvalcore/cmor/_fixes/native6/era5.py

```diff
@@ -24,17 +27,23 @@ def get_frequency(cube):
 
     time.convert_units("days since 1850-1-1 00:00:00.0")
     if len(time.points) == 1:
-        if cube.long_name != "Geopotential":
+        acceptable_long_names = (
+            "Geopotential",
+            "Percentage of the Grid Cell Occupied by Land (Including Lakes)",
+        )
+        if cube.long_name not in acceptable_long_names:
             raise ValueError(
                 "Unable to infer frequency of cube "
                 f"with length 1 time dimension: {cube}"
             )
         return "fx"
 
     interval = time.points[1] - time.points[0]
+
     if interval - 1 / 24 < 1e-4:
         return "hourly"
-
+    if interval - 1.0 < 1e-4:
+        return "daily"
```

Comment:
Why is this code needed? ERA5 data is always either hourly or monthly means as far as I'm aware

Reply:
Fixed in https://github.com/ESMValGroup/ESMValCore/pull/2178/commits/a031ea9516ce1db269396d46e380fad89d1191bd

Reply:
The ERA5 GRIB data on Levante is also available in daily format (`1D`), see also https://docs.dkrz.de/doc/dataservices/finding_and_accessing_data/era_data/index.html#era5-data-via-pool-data-era5.